internal/statesync/snapshots.go

```diff
@@ -80,7 +80,7 @@ func newSnapshotPool(stateProvider StateProvider) *snapshotPool {
 // snapshot height is verified using the light client, and the expected app hash
 // is set for the snapshot.
 func (p *snapshotPool) Add(peerID types.NodeID, snapshot *snapshot) (bool, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancel := context.WithTimeout(context.TODO(), 30*time.Second)
```

Comment:
Calls to stateProvider.Commit() and stateProvider.State() have a 10 second timeout. I think these calls and the AppHash should be roughly the same time (if not more for the former) given the amount of network io

Reply:
It looked to me like this just gave it more time to run through its retry loops, which seems like a good thing?

Reply:
Yeah I don't see it harming state sync. I was just saying that we should probably update the other stateProvider calls to have  a 30 second timeout for consistency

Reply:
where are those?

Reply:
grep `.Commit(` in statesync :)

Reply:
I think it's in `syncer.go#Sync()`

Reply:
I think I got these.